osu.Game.Rulesets.Catch/Objects/Drawables/DrawablePalpableCatchHitObject.cs

```diff
@@ -31,12 +43,42 @@ protected DrawablePalpableCatchHitObject(CatchHitObject h)
                 Origin = Anchor.Centre,
                 Anchor = Anchor.Centre,
             });
+
+            IndexInBeatmap.Value = h.IndexInBeatmap;
```

Comment:
This is a bit weird, why is this done here instead of relying on the binding in `OnApply`?

Reply:
It can be removed. It was for testing (when DHO is not placed in a playfield and not receiving `OnApply`) but it is not needed for testing anymore.

Reply:
Yeah it should be removed then imo, just for consistency. The passing of properties is going to be a part of the eventual wiki on pooling so there should be a clear direction for how to do this stuff (e.g. only via `OnApply()`).